src/components/navigation/search-input.tsx

```diff
@@ -65,6 +65,32 @@ export function SearchInput({ onSearchPage }: SearchInputProps) {
       input.setAttribute('placeholder', strings.header.search)
       input.focus()
       setSearchActive(true)
+
+      const script = document.createElement('script')
```

Comment:
have you tried to include your script directly (without the script tag?). I think, this *should* work.

Reply:
oh wow. obviously way nicer.